### ISSUE TITLE
Generic country flag

### ISSUE THREAD

**Issue by derwaldgeist** [Templarian/MaterialDesign#2374]
There are quite a lot of flag symbols in the font already, but all of them contain a flag pole. I was looking for a generic country flag *without* a pole, to represent countries as a category. I would like to propose to take the "flag-checkered-variant", but remove its pole.

**Comment by MrGrigri** [comment]
I would suggest using the `earth` icon.

![earth](https://materialdesignicons.com/icon/earth)

**Comment by derwaldgeist** [comment]
This doesn’t really work for my case. I have a separation between cities, countries, and POIs. So earth would comprise them all

**Comment by JamesCoyle** [comment]
Can you share the UI you are working on? Might help us get a better idea.

**Comment by derwaldgeist** [comment]
Sure. I need it for a list of places, in which we will have three categories of places: points of interest, cities, and countries. For POIs and cities, I already found an icon, but for the "countries" category I couldn't find a matching one.

![image](https://user-images.githubusercontent.com/3984535/30018289-9c30b01a-915c-11e7-9398-4e1d63f928c2.png)

**Comment by Ch-Lo** [comment]
Why does [flag-variant](https://materialdesignicons.com/icon/flag-variant) not work?

**Comment by derwaldgeist** [comment]
Well, I am using such an icon at the moment. Because I would prefer to get rid of the pole.

**Comment by MrGrigri** [comment]
I agree, `flag-variant` should be the answer. Without a pole, the icon would resemble just cloth and not a flag.

**Comment by goyney** [comment]
Another issue has been referenced here looking for a "towel," probably because taking the flag pole off a flag really makes no sense. :)

However, a towel, does make sense, so I'm updating this issue to be a request for a towel, or a wavy-piece of cloth.
